R/layer_population_scaling.R

```diff
@@ -156,9 +157,10 @@ slather.layer_population_scaling <-
     ) %>%
       dplyr::mutate(dplyr::across(
         dplyr::all_of(col_names),
-        ~.x * !!pop_col / object$rate_rescaling ,
-        .names = "{.col}{suffix}")) %>%
-     dplyr::select(-!!pop_col)
+        ~ .x * !!pop_col / object$rate_rescaling,
+        .names = "{.col}{suffix}"
+      )) %>%
+      dplyr::select(-any_of(col_to_remove))
```

Comment:
I suspect that `any_of()` needs a package prefix: `dplyr::any_of()`. I don't think we've imported it elsewhere.